src/constants/solidity/code-exercises.ts

```diff
@@ -300,4 +300,8 @@ export const CODE_EXERCISES: CodeTemplatesType = {
     exercise5:
       "// SPDX-License-Identifier: MIT\npragma solidity ^0.8.24;\n\n// Step 1: Implement a contract named 'SendEtherChecked'\n// Step 2: Inside the 'SendEtherChecked' contract, declare a public mapping named 'balances' that maps addresses to uint256\n// Step 3: Still inside the 'SendEtherChecked' contract, implement a function named 'sendEther' that takes two parameters: an address payable 'recipient' and a uint256 'amount'\n// Step 4: The 'sendEther' function should be public and payable\n// Step 5: Inside the 'sendEther' function, add a require statement to check that the sender has enough Ether to send\n// Step 6: Still inside the 'sendEther' function, update the balances of the sender and the recipient\n// Step 7: Still inside the 'sendEther' function, use the 'call' method to send the specified amount of Ether to the specified address\n// Step 8: Add a require statement to check that the call was successful\n",
   },
+  "transient-storage": {
```

Comment:
I added 1 but 5 exercises might be redundant since the transient storage only uses 2 opcodes (`tstore` and `tload`). You can only asign values to keys and load them later.

Regarding those elements, please tell me if you still want me to implement 2 more exercises

Reply:
Agreed! But yes, would still be great if we add to 5 exercises. Reason being, the existing lesson architecture is designed to capture 5 pair of exercise/solution to fortify learning through repetition. Otherwise, the client would also bug out if there's empty exercises/solutions :( 

My recommendation here is to repeat `exercise1` and `exercise2` but with a slight tweak. For example, storing a different data type like `string` instead of `uint256` or maybe even declaring a different variable. Mini tweaks like these would help newer devs learn while having that familiarity from the earlier exercises.

Reply:
I'll implement that